modules/net-address/README.md

```diff
@@ -47,6 +47,23 @@ module "addresses" {
 # tftest:modules=1:resources=2
 ```
 
+### PSA addresses
+
+```hcl
+module "addresses" {
+  source     = "./modules/net-address"
+  project_id = var.project_id
+  psa_addresses = {
+    cloudsql-mysql = {
+      address       = "10.10.10.0"
```

Comment:
Would it make sense to use `range` so as to avoid the need for an extra attribute?

Reply:
Makes sense - the reasoning behind not doing that was to keep a the interface consistent with psc_addresses and the resource itself. It's a trivial change, your call.

Reply:
I'm all for simplifying the interface (cf the GKE module for extreme examples of the approach). It's a minor thing, so it's also ok to leave it like this.